apps/api/src/core/services/hono-error-handler/hono-error-handler.service.ts

```diff
@@ -33,7 +33,11 @@ export class HonoErrorHandlerService {
       return c.json({ error: "BadRequestError", data: error.errors }, { status: 400 });
     }
 
-    await this.reportError(error, c);
+    try {
```

Comment:
what would you think of handling this in the `reportError` method itself. Seems like it's more relevant there and this method is cleaner